question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -128,11 +128,33 @@ def inner_html(element):
     return html
 
 
+def compat_has(object, attrib):
+    if attrib in object:
+        return True
+    new_attrib = attrib.replace('-', '_')
+    return new_attrib in object
+
+
+def compat_get(object, attrib, default):
+    if attrib in object:
+        return object[attrib]
+    new_attrib = attrib.replace('-', '_')
```

Comment:
I think the overall logic here is correct, but shouldn't this be called `old_attrib` rather than `new_attrib`? As in, this variable is going to be in the "old" (i.e., with underscores) format, right? Same comment applies to `compat_has()` above.

Reply:
Good call, I think I had the logic reversed at first. Will fix.